tests/php/ORM/DataListTest.php

```diff
@@ -865,9 +865,7 @@ public static function provideDefaultSort(): array
         ];
     }
 
-    /**
-     * @dataProvider provideDefaultSort
-     */
+    #[DataProvider('provideDefaultSort')]
```

Comment:
Unrelated MNT fix. Noticed a warning about this in the test run so I figured I may as well just get it fixed.